server/bleep/src/intelligence/language/python/mod.rs

```diff
@@ -159,10 +159,6 @@ mod tests {
                     imports: [
                         List {
                             context: "from typings import §List§",
```

Comment:
Why has this been removed?

Reply:
This is an expect test, this change is a result of upgrading the upstream deps.

Reply:
these tests shouldn't change on version upgrade. will take a look at what is going on here.

Reply:
can we roll-back the change to python version dependency? it contains a breaking change to the grammar.